packages/shared/src/components/drawers/NavDrawerItem.tsx

```diff
@@ -4,7 +4,7 @@ import { Button, ButtonProps, ButtonVariant } from '../buttons/Button';
 import ConditionalWrapper from '../ConditionalWrapper';
 
 export interface NavItemProps
-  extends Pick<ButtonProps<'a'>, 'href' | 'icon' | 'onClick'> {
+  extends Pick<ButtonProps<'a'>, 'href' | 'icon' | 'onClick' | 'target'> {
```

Comment:
Wondering if we should just accept all buttonProps?
For instance when working with links you'll soon need `target` and `rel` as well.

Reply:
Yeah, I tried and failed. TS was complaining if I just used `<ButtonProps<'a'>` for some reason. This seemed quicker than trying to figure out the type system complaints...